src/components/Login.jsx

```diff
@@ -4,11 +4,8 @@ import { useNavigate } from 'react-router-dom'
 import useDerivAccounts from '../hooks/useDerivAccounts'
 
 const Login = () => {
-    const APP_ID = '9999'
-    // Force http for development, use normal origin for production
-    const REDIRECT_URI = import.meta.env.DEV
-        ? `http://${window.location.host}`
-        : window.location.origin
+    const APP_ID = '66435'
+    const REDIRECT_URI = 'https://ashkan-deriv.github.io'
```

Comment:
it should be **https://ashkan-deriv.github.io/copy-trading**
also I noticed a redirect to https://ashkan-deriv.github.io/login. where is it coming from? can we remove it?

Reply:
This redirect_uri is when you successfully logs in, it will redirect back to this url which in our case is https://ashkan-deriv.github.io/copy-trading.

Reply:
ok. so it should be https://ashkan-deriv.github.io/copy-trading, correct?